src/inputs/ChipSelectField.stories.tsx

```diff
@@ -142,3 +143,34 @@ export function Example() {
     </div>
   );
 }
+
+export function PerfTest() {
```

Comment:
praise: This is cool! I was wondering about options for failing the build on perf regressions to GridTable. Does this do that?

Reply:
I didn't see a way to store the results of the performance testing in the application so that we can fail on regressions. We can dig in further and see what we might get there.